.gitignore

```diff
@@ -10,7 +10,8 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
```

Comment:
dropped it since it seems to me irrelevant right now but if you want to keep, I can revert it.